src/sdkToEventsApiConverter.ts

```diff
@@ -653,8 +661,9 @@ export function convertUserAttributeChangeEvent(
 export function convertUserIdentityChangeEvent(
     sdkEvent: SDKEvent
 ): EventsApi.UserIdentityChangeEvent | null {
-    const commonEventData: EventsApi.CommonEventData =
-        convertBaseEventData(sdkEvent);
+    const commonEventData: EventsApi.CommonEventData = convertBaseEventData(
+        sdkEvent
+    );
```

Comment:
nit: I feel like these linting issues from auto-linting on save shoudln't be part of this commit, but I also don't really care that much.  I just feel like it detracts from the PR.  We can talk about that though.